pkg/controlplane/reconcilers/lease.go

```diff
@@ -67,6 +67,7 @@ func (s *storageLeases) ListLeases() ([]string, error) {
 		ResourceVersion:      "0",
 		ResourceVersionMatch: metav1.ResourceVersionMatchNotOlderThan,
 		Predicate:            storage.Everything,
+		Recursive:            true,
```

Comment:
this is a good example of the usage change that is surprising/unfortunate

Reply:
Renamed the new function to something new, so that every usage has to be explicitly migrated.